src/main/java/by/exadel/internship/controller/UserController.java

```diff
@@ -27,9 +25,16 @@
         return userService.getAll();
     }
 
-    @GetMapping("/{userId}")
+    @GetMapping("/userId")
```

Comment:
userId should be query parameter, not part of the path

Reply:
sory, maybe I didn't fully understand what you meant about all the cases, now I have fixed everything, done

Reply:
Why do you leave /userId as part of the path? 
Please review https://medium.com/@mwaysolutions/10-best-practices-for-better-restful-api-cbe81b06f291 once again